src/main/java/com/ombremoon/enderring/common/init/entity/StatusEffectInit.java

```diff
@@ -71,39 +72,40 @@ public class StatusEffectInit {
                                     "GREENSPILL_CRYSTAL",
                                     10F, AttributeModifier.Operation.ADDITION),
                             EpicFightAttributes.MAX_STAMINA
-                    ).build());
+                    )
+                    .addTier("f8b61f3c-04cc-64b3-6b08-67b8bdb40834",1,0.15f).build());
 
     public static final RegistryObject<MobEffect> GREENBURST_CRYSTAL = registerSimpleEffect("greenburst_crystal_tear",
             new EffectBuilder(EffectType.STAMINA_REGEN)
                     .addAttribute(new AttributeModifier(
                                     UUID.fromString("f91d1f3c-04cc-64b3-6b08-67b8bdb40834"),
                                     "GREENBURST_CRYSTAL",
-                                    15F, AttributeModifier.Operation.MULTIPLY_BASE),
+                                    0.15F, AttributeModifier.Operation.MULTIPLY_BASE),
```

Comment:
Is supposed to be a flat increase to stamina recovery. All stamina numbers are divided by 15 though, so if the wiki says "increases by 15", in the our math, that means "increases by 1"